File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/backup/BackupModel.java
package com.raizlabs.android.dbflow.test.structure.backup;

import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;

/**
 * Description:
 */
@Table(database = BackupDatabase.class)
public class BackupModel extends BaseModel {

    @PrimaryKey
    String name;

    public String getName() {
        return name;
    }
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/structure/backup/BackupModel.kt
package com.raizlabs.android.dbflow.structure.backup

import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel

/**
 * Description:
 */
@Table(database = BackupDatabase::class)
class BackupModel : BaseModel() {

    @PrimaryKey
    var name: String? = null
}
